package.json

```diff
@@ -223,6 +224,11 @@
                 "type": "boolean",
                 "description": "Whether the app should be rebuild before start.",
                 "default": true
+              },
+              "syncAllFiles": {
```

Comment:
yep we can rename this before releasing. to avoid "braking change" it later

@hshristov